src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +19,14 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
```

Comment:
```suggestion
        return numbers.stream()
```
це не помилка, але в більшості випадків `.stream()` пишуть на тому ж рядку